docs/reference/modules/cluster/remote-clusters-connect.asciidoc

```diff
@@ -0,0 +1,222 @@
+[[remote-clusters-connect]]
+=== Connect to remote clusters
+To connect a remote cluster (Cluster A) to a local cluster (Cluster B), you
+configure Cluster A as a remote on Cluster B.
+
+image::images/connect-remote-clusters.png[ClusterA contains the leader index and ClusterB contains the follower index]
+
+To configure a remote cluster from Stack Management in {kib}:
+
+. Select *Remote Clusters* from the side navigation.
+. Specify the {es} endpoint URL, or the IP address or host name of the remote
+cluster (`ClusterA`) followed by the transport port (defaults to `9300`). For
+example, `cluster.es.eastus2.staging.azure.foundit.no:9400` or
+`192.168.1.1:9300`.
+
+Alternatively, use the <<cluster-update-settings,cluster update settings API>>
+to add a remote cluster. You can also use this API to 
+<<configure-remote-clusters-dynamic,dynamically configure>> remote settings on
+every node in the cluster:
+
+[source,console]
+----
+PUT /_cluster/settings
+{
+  "persistent" : {
+    "cluster" : {
+      "remote" : {
+        "leader" : {
```

Comment:
I'm clarifying and expanding the lead-in to this example by saying:

> The following request adds a remote cluster with an alias of `leader`. This _cluster alias_ represents the connection to the remote cluster and is used  subsequently to distinguish between local and remote indices. When adding a remote cluster in {kib}, the cluster alias serves as a unique identifier for the remote cluster.

Reply:
I wonder whether we should avoid using `leader` for the remote cluster. The word `leader` has a specific meaning in CCR for the leader indices. While it kinda makes sense to also call the cluster `leader` for a CCR setup, it does not quite fit for a CCS setup because there is no leader indices for CCS.

Reply:
Agreed -- I had copied this example over from the CCR docs, but it should be updated to match the surrounding text. I'll change `leader` to `cluster_one` so that it matches the other examples in this section.